PF1/Assignment6/TestAssignment.java

```diff
@@ -0,0 +1,70 @@
+package assignment6;
+import org.junit.Assert;
+
+
+import org.junit.Test;
+
+public class TestAssignment {
+	//Test case if both array are in ascending order
+	@Test
+	public void mtest1()
+	{
+		JoinSortedArray sort_array =new JoinSortedArray();
+		int a[]={5,7,9,10};
+		int b[]={4,6,8,11};
+		int c[]=new int[a.length+b.length];
+		int result_array[]={4,5,6,7,8,9,10,11};
+		c =sort_array.join(a, a.length, b, b.length, c);
+		Assert.assertArrayEquals(result_array,c);
+	}
+	//test case if all elements of one array is same
+	 @Test
+	 public void mtest2()
+		{
+			JoinSortedArray sort_array =new JoinSortedArray();
```

Comment:
discrepancy in test class implementations. used @before in all the assignments except this last one.